README.md

```diff
@@ -35,7 +35,7 @@ Then, set it up:
 
 ### Upgrading from an older version
 
-- When upgrading to version 4.0.0 you may now take advantage of an in-memory cache
+- When upgrading to version 4.1.0 you may now take advantage of an in-memory cache
```

Comment:
The cache was introduced in 4.0.0 and that is still true, right?

Reply:
It is, I'll rephrase